src/test/suite/fixtures/recorded/languages/rust/chuckArgTrap2.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
+command:
+  spokenForm: chuck arg trap
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: t}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    enum Maybe<T> {
+        Just(T),
+        Nothing,
+    }
+  selections:
+    - anchor: {line: 4, character: 0}
+      active: {line: 4, character: 0}
+  marks:
+    default.t:
+      start: {line: 1, character: 9}
+      end: {line: 1, character: 10}
+finalState:
+  documentContents: |
+    enum Maybe<T> {
+        Just(),
```

Comment:
if you're doing a "chuck" test, probably worth having two args so that you can test comma removal

Reply:
Yeah, I’ve tested this manually, but couldn’t come up with an example of an enum with more than one type arguments which are used on a single data constructor.

Reply:
Sure, I just wanted to come-up with something meaningful for no reason

Reply:
> Couldn't you just do something like this?
> 
> ```rust
> enum Foo {
>     Bar(String, String),
>     Baz,
> }
> ```

I’ll add a separate test for that I think.

Reply:
> Well if you want something meaningful, the Rust docs use the following:
> 
> ```rust
> enum IpAddr {
>     V4(u8, u8, u8, u8),
>     V6(String),
> }
> ```
> 
> But I'm not sure why it needs to be meaningful; it's just testing syntactic structures 🤷

Oh yeah, excellent example.